.gitignore

```diff
@@ -1,152 +0,0 @@
-!.cruft.json
-!.yamllint.yaml
-!.flake8
-!.mypy.ini
-!.editorconfig
-!.gitignore
```

Comment:
Why did you delete this?

Reply:
Got a bit too greedy of removing stuff. I agree this should stay and I see it's common in other repos e.g. https://github.com/pandas-dev/pandas/blob/main/.gitignore and https://github.com/tiangolo/fastapi/blob/master/.gitignore